modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMDataProviderInstancesServlet.java

```diff
@@ -105,8 +104,7 @@ private JSONArray _getDataProviderInstancesJSONArray(
 				scopeGroupId = scopeGroup.getLiveGroupId();
 			}
 
-			long[] groupIds = _portal.getCurrentAndAncestorSiteGroupIds(
-				scopeGroupId);
+			long[] groupIds = {scopeGroupId};
```

Comment:
@icgsbr we should keep the current behavior to avoid regression bugs.. I thought about doing something like:

```